src/_layouts/post.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  {% include head.html %}
+
+  <body>
+    {% include header.html %}
+
+    <main>
+      <div id="main_inner">
+        {% if page.title %}
+        <h2>{{ page.title }}</h2>
+        {% endif %}
```

Comment:
Can we get the date from the blog post in any way (so that we can show it on the post, and probably also on the blogs page so people can see at a glance how old each blog post is)?

Reply:
Good question. I will try to find out.

Reply:
Done. Do you think the list of blogs is OK - I wonder if it is a bit clunky.

Reply:
Looks fine to me- a little basic if anyone wanted to put this in a portfolio for a design job at a big company, but it loads quickly and shows what it needs to show so I'm inclined to leave it as is unless accessibility concerns arise, or unless it gets unwieldy when we have $number_of_blog_posts_we_end_up_with.
I'd have ended up with the same sort of appearance if it was me, I think. [smiles]